pkg/testutil/testutil.go

```diff
@@ -45,3 +46,13 @@ func MustNewURL(t *testing.T, s string) *url.URL {
 	}
 	return u
 }
+
+// FatalStack prints out error message of stack trace and fatals.
+// This is useful in case tests times out because of slow I/O, etc.
```

Comment:
let's remove the timeout thing. FatalStack helps to fatal the test and print out the stacks of all running goroutines.